xclim/utils.py

```diff
@@ -671,7 +672,7 @@ def __call__(self, *args, **kwds):
                 attrs[attr] += "{}: ".format(p) if self._nvar > 1 else ""
                 attrs[attr] += getattr(ba.arguments[p], attr, '')
                 if attrs[attr]:
-                    attrs[attr] += "\n" if attr == 'history' else " "
+                    attrs[attr] += ": xclim version: {} ".format(xclim.__version__) if attr == 'history' else " "
```

Comment:
No. Put the version with the function's call signature around line 693. 
Also add a test.